src/app/pages/jails/jail-list/jail-list.component.ts

```diff
@@ -116,20 +116,26 @@ export class JailListComponent implements OnInit {
       enable: true,
       ttpos: "above",
       onClick: (selected) => {
-        let selectedJails = this.getSelectedNames(selected);
-        this.loader.open();
+        const selectedJails = this.getSelectedNames(selected);
+        this.snackBar.open(T('Update selected plugins started.'), 'close', { duration: 5000 });
         this.entityList.busy =
           this.ws.job('core.bulk', ["jail.update_to_latest_patch", selectedJails]).subscribe(
             (res) => {
-              this.loader.close();
-              if (res.state == 'SUCCESS') {
-                this.snackBar.open(T("Selected Jail(s) updated successfully."), T("Close"), { duration: 5000 });
+              let message = "";
+              for (let i = 0; i < res.result.length; i++) {
+                if (res.result[i].error != null) {
+                  message = message + '<li>' + selectedJails[i] + ': ' + res.result[i].error + '</li>';
+                }
+              }
+              if (message === "") {
+                this.snackBar.open(T('Update selected jails succeeded.'), 'close', { duration: 5000 });
               } else {
-                new EntityUtils().handleWSError(this.entityList, res, this.dialogService);
+                message = '<ul>' + message + '</ul>';
+                this.dialogService.errorReport(T('Update following selected jails failed.'), message);
```

Comment:
It's hard to tell what "following" means here. Is it a list of all the jails that failed to update?  Does it include the specific errors for each jail?  Or is there just one message that says all of those jails failed?  "selected" doesn't help much, the user might not remember which they selected.  A clearer message would have the important part first: "Failed to update these jails:" and then the list of jails and hopefully a clear error for each:
jail1 (failed to fetch)
jail2 (out of space)
...

Reply:
We do list the error for each item. Please see the screenshot send on ryver.